### PR TITLE
ICU-21389 Merge maint/maint-68 to master

### PR DESCRIPTION
**Pull request by sffc**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21389
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

**Comment by sffc** [comment]
As discussed in ICU-TC: Wait for #1461 until Friday; if #1461 is not finished by Friday, then merge this PR without it.

**Comment by jefgen** [comment]
Note: There is one more PR for `maint/maint-68` branch, #1465, which should hopefully be merged today.

**Comment by sffc** [comment]
I think all open PRs against maint/maint-68 are merged finally.  I will merge this PR to master as soon as I get a stamp of approval.

**Review by jefgen — APPROVED** [review]

**Pull request merged**
